tools/nix/parts/tooling.nix

```diff
@@ -6,10 +6,21 @@ _: {
     system,
     ...
   }: {
+    packages.uploadserver = pkgs.python311Packages.buildPythonPackage rec {
+      pname = "uploadserver";
+      version = "5.2.0";
+      src = pkgs.python311Packages.fetchPypi {
+        inherit pname version;
+        sha256 = "sha256-M2gHbyj7HAbe0nuQeV5fwuejend5/Ksb15oaUzQXUfU=";
+      };
+      doCheck = false; # Disable tests
+    };
+
```

Comment:
@SanjoDeundiak @adrianbenavides we now bake uploadserver in our Nix configuration